src/Tools/dotnet-monitor/CollectionRules/Options/CollectionRuleOptions.cs

```diff
@@ -37,6 +37,18 @@ internal sealed partial class CollectionRuleOptions
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_CollectionRuleOptions_Limits))]
         public CollectionRuleLimitsOptions? Limits { get; set; }
 
-        internal List<ValidationResult> ErrorList { get; } = new List<ValidationResult>();
+        internal List<ErrorValidationResult> ErrorList { get; } = new List<ErrorValidationResult>();
```

Comment:
Could you add a source comment explaining why `ValidationResult` cannot be used here?